hw02/EnemyCollision.cs

```diff
@@ -18,39 +18,74 @@ void Update()
     
     // TODO: Problem 7: Why did I not include the words public or private here?
     // What does it mean when I only write void (Movement)? What does void mean?
-    void Movement ()
+    void Movement () // "void" is used to specify a type of function that does NOT return a value
```

Comment:
That's correct! And without specifying public or private, it will be private in default.